templates/csharp/app-mvvm/App.axaml

```diff
@@ -10,3 +11,19 @@
         <FluentTheme Mode="Light"/>
     </Application.Styles>
 </Application>
+//#else
+<Application xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:local="using:AvaloniaAppTemplate"
+             x:Class="AvaloniaAppTemplate.App"
+             RequestedThemeVariant="Default">
+             <!-- "Default" ThemeVariant follows system theme for Themes that support it. -->
```

Comment:
```suggestion
             <!-- "Default" ThemeVariant follows system theme variant. "Dark" or "Light" are other available options. -->
```
Third party themes won't support it immediately, but we can resolve these questions when they come.